mantidimaging/gui/windows/spectrum_viewer/spectrum_widget.py

```diff
@@ -410,3 +413,24 @@ def set_join_plot(self) -> None:
                     item.setPen(None)
                     item.setSymbol('o')
                     item.setSymbolBrush(item.original_pen)
+
+    def set_right_anchored_child(self, child: None | GraphicsItem, offset: tuple[int, int]) -> None:
+        """
+        Anchor a child object to the top right of this item
+
+        @param child: child object
+        @param offset: tuple (x,y) offset from the top right
+        """
+        self.anchored_child = child
+        self.anchored_offset = offset
+        self.update_anchor()
+
+    def resizeEvent(self, event) -> None:
```

Comment:
The event argument should have an annotation
`def resizeEvent(self, event: QResizeEvent) -> None:`